canonical-web-and-design/jp.ubuntu.com
# News on frontpage

## Done
Fixes https://github.com/ubuntudesign/base-squad/issues/453
Fixes https://github.com/ubuntudesign/base-squad/issues/472
Fixes https://github.com/ubuntudesign/base-squad/issues/473
- Adds new JavaScript build pipeline
- Fixes global-nav building
- Removes unused dependency
- Adds last 3  articles in japanese and last highlighted article onto the homepage


## QA

- ./run
- go to `http://localhost:8012` and make sure that 3 highlighted articles appear.
- no spotlight article exists in wordpress so this wont be shown yet

## DESIGN

Right now the news are just right on the top. Where should I place this?

## Screenshot
Here is a screenshot with a fake highlighted article

![image](https://user-images.githubusercontent.com/2843450/55948562-b7a5ae00-5c48-11e9-9aa4-0d45da7df786.png)
